.github/workflows/phpunit.yml

```diff
@@ -26,11 +26,19 @@ jobs:
       with:
         php-version: '7.4'
         coverage: none
-        tools: composer:v1
+        tools: composer:v2
 
     - name: Install dependencies
       run: composer install
 
+    - name: install node v16
+      uses: actions/setup-node@v1
+      with:
+        node-version: 16
```

Comment:
Looks like Node 18 (latest) is already used successfully in other workflows. Do we even need this lower version here?

Reply:
@cadic as discussed yesterday on Slack, leaving this as is.
As most of our projects run on 16, we can upgrade this later when we bump up across projects.